onnxscript/rewriter/ort_fusions/bias_gelu_test.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import unittest
+
+import numpy as np
+import onnx
+
+import onnxscript.ir as ir
+import onnxscript.rewriter.ort_fusions._test_utils as test_utils
+from onnxscript.optimizer import optimize, remove_unused_nodes
+from onnxscript.rewriter.ort_fusions.bias_gelu import fuse_bias_gelu
+
+FLOAT = onnx.TensorProto.FLOAT
+
+
+class BiasGeluFusionTest(unittest.TestCase):
+    def test_gelu_fusion(self):
+        model_proto = onnx.helper.make_model(
```

Comment:
nit: Would be simpler to use onnxscript (1st choice) or onnx parser (2nd choice) to construct models. (I guess this may be coming from a pre-existing test?)

Reply:
Modified to use onnxscript style test